sim/train_tdmpc.py

```diff
@@ -0,0 +1,208 @@
+"""Trains a humanoid to stand up."""
```

Comment:
We can keep this separate until some walking results will be achieved but later down the line it should be combined with the base train logic.